test/functional/sqlsrv/test_non_alpha_password.phpt

```diff
@@ -51,5 +62,5 @@ print "Test successful";
 ?> 
 --EXPECTREGEX--
 An unescaped right brace \(}\) was found in either the user name or password.  All right braces must be escaped with another right brace \(}}\)\.
-Warning: sqlsrv_close\(\) expects parameter 1 to be resource, boolean given in .+(\/|\\)test_non_alpha_password\.php on line 38
+Warning: sqlsrv_close\(\) expects parameter 1 to be resource, boolean given in .+(\/|\\)test_non_alpha_password\.php on line 50
```

Comment:
Do we want to specify a line number? Maybe make the number a regex? Or maybe put in a conditional like `if ($conn) sqlsrv_close($conn);` so that we don't output this warning at all  when the test is working properly, otherwise seeing this message in a diff or log file makes it look at first glance like there's a problem.